modules/apps/kate/default.nix

```diff
@@ -196,25 +196,65 @@ in
     '';
   };
 
+  # ==================================
+  #     BRACKETS
+  options.programs.kate.editor.brackets = {
+    characters = lib.mkOption {
+      type = lib.types.str;
+      default = "<>(){}[]'\"\`";
+      example = "<>(){}[]'\"\`*_~";
+      description = "Which characters kate will treat as brackets.";
+    };
+    automaticallyAddClosing = lib.mkEnableOption {
+      description = "that a closing bracket is automatically inserted upon typing the opening";
+    };
+    highlightRangeBetween = lib.mkEnableOption {
+      description = ''
+        that the range between the brackets is highlighted when the cursor is adjacent
+        to any of the brackets
+      '';
+    };
+    highlightMatching = lib.mkEnableOption {
+      description = ''
+        that the first two lines of the matching opening bracket are shown in a small floating window
+        when the cursor is next to the closing bracket and the opening bracket is not in the visible
+        area
+      '';
+    };
+    flashMatching = lib.mkEnableOption {
+      description = ''
+        that the matching bracket flashes when the cursor moves adjacent to the other
+      '';
+    };
+  };
+
   # ==================================
   #     WRITING THE KATERC
   config.programs.plasma.configFile."katerc" = lib.mkIf cfg.enable {
     "KTextEditor Document" = {
-      "Auto Detect Indent" = cfg.editor.indent.autodetect;
-      "Indentation Width" = cfg.editor.indent.width;
+      "Auto Detect Indent".value = cfg.editor.indent.autodetect;
+      "Indentation Width".value = cfg.editor.indent.width;
```

Comment:
Can drop `.value`. It doesn't matter, but after they were made optional the rest of the project doesn't use them unless you need to set `immutability` or something similar for the key.